### ISSUE TITLE
[Edit] Split all links by language

### ISSUE THREAD

**Issue by Sepera-okeq** [anshumanv/awesome-anime-sources#60]
Why not sort by language? I think this would not be a bad thing and would not make confusion in the future, when there will be even more links

I have this idea: Mark the country's flag before the link (for example, EN is the flag of Britain, and if Russian is the flag of Russia, and so on)

**Comment by anshumanv** [comment]
Yeah flag makes sense to me, also I think we can keep specific languages at the bottom of each section to group them all in once place.

views @toprak?

**Comment by toprak** [comment]
It sounds good, I do not have any idea how to do that though. I will have break tomorrow, I will try my best.

**Comment by anshumanv** [comment]
You can paste the country flag as markdown and it should work

**Comment by toprak** [comment]
@anshumanv i wanna do section like you click a flag and it shows just filtered language but could not find a way to do like that. do you have any idea brother

**Comment by anshumanv** [comment]
yeah sounds good to me

**Comment by toprak** [comment]
> yeah sounds good to me

yeah it sounds good but i dont know how to do it also could not find a way to manage that way need ur help :D

**Comment by anshumanv** [comment]
Let me find a good solution in a bit!

**Comment by anshumanv** [comment]
Let me find a good solution in a bit!

**Comment by AkhshyBalakannan** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @Jakkc, @zerob13, @sarfraznawaz2005, @p0123n, @youanden, @eueung, @vinhup, @Leon555, @gabeio, @tonyfast, @Samircj, @snorik, @NPavl, @ququzone, @thomasthiebaud, @adamclark64, @Nektarium, @floriangosse, @Multiboxer, @jsoetens
